app/src/main/java/org/oppia/app/recyclerview/DragItemTouchHelperCallback.kt

```diff
@@ -0,0 +1,64 @@
+package org.oppia.app.recyclerview
+
+import androidx.recyclerview.widget.ItemTouchHelper
+import androidx.recyclerview.widget.RecyclerView
+
+/**
+ * This is used to enable functionality like  drag up and down to the RecyclerView.
+ */
+class DragItemTouchHelperCallback private constructor(dragDirs: Int, swipeDirs: Int) :
```

Comment:
We should name this based on what it does. 'helper' and 'callback' are a bit vague in this context. Maybe ``DragAndDropItemFacilitator`` or similar.

Reply:
done